crates/sui/src/validator_commands.rs

```diff
@@ -454,6 +467,66 @@ impl SuiValidatorCommand {
                     serialized_data,
                 }
             }
+            SuiValidatorCommand::BridgeCommitteeRegistration {
+                bridge_node_config_path,
+                bridge_authority_url,
+            } => {
+                let bridge_config = BridgeNodeConfig::load(bridge_node_config_path)
+                    .expect("Couldn't load BridgeNodeConfig");
+
+                // Read bridge keypair
+                let key_str =
+                    fs::read_to_string(bridge_config.bridge_authority_key_path_base64_raw)?;
+                let bytes = Base64::decode(key_str.trim()).unwrap();
+                let key = Secp256k1PrivateKey::from_bytes(&bytes).unwrap();
+                let ecdsa_keypair = Secp256k1KeyPair::from(key);
+
+                // Read sui node account key pair
+                let address = context.active_address()?;
+                println!("Starting bridge committee registration for Sui validator: {address}, with bridge public key: {}", ecdsa_keypair.public);
+
+                let bridge_client =
+                    sui_bridge::sui_client::SuiClient::new(&bridge_config.sui.sui_rpc_url).await?;
+                let bridge = bridge_client
+                    .get_mutable_bridge_object_arg_must_succeed()
+                    .await;
+
+                let gas = context
+                    .get_one_gas_object_owned_by_address(address)
+                    .await?
+                    .unwrap();
+                let ref_gas_price = context.get_reference_gas_price().await?;
```

Comment:
nit: we use `gas_price` everywhere in this codebase, so I'd suggest we keep it consistent.
```suggestion
                let gas_price = context.get_reference_gas_price().await?;
```